docs/source/examples/triso.ipynb

```diff
@@ -0,0 +1,378 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "OpenMC includes a few convenience functions for generationing TRISO particle locations and placing them in a lattice. To be clear, this capability is not a stochastic geometry capability like that included in MCNP. It's also important to note that OpenMC does not use delta tracking, which would normally speed up calculations in geometries with tons of surfaces and cells. However, the computational burden can be eased by placing TRISO particles in a lattice."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from math import pi\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import openmc\n",
+    "import openmc.model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's first start by creating materials that will be used in our TRISO particles and the background material."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fuel = openmc.Material(name='Fuel')\n",
+    "fuel.set_density('g/cm3', 10.5)\n",
+    "fuel.add_nuclide('U235', 4.6716e-02)\n",
+    "fuel.add_nuclide('U238', 2.8697e-01)\n",
+    "fuel.add_nuclide('O16',  5.0000e-01)\n",
+    "fuel.add_element('C', 1.6667e-01)\n",
+    "\n",
+    "buff = openmc.Material(name='Buffer')\n",
+    "buff.set_density('g/cm3', 1.0)\n",
+    "buff.add_element('C', 1.0)\n",
+    "buff.add_s_alpha_beta('c_Graphite')\n",
+    "\n",
+    "PyC1 = openmc.Material(name='PyC1')\n",
+    "PyC1.set_density('g/cm3', 1.9)\n",
+    "PyC1.add_element('C', 1.0)\n",
+    "PyC1.add_s_alpha_beta('c_Graphite')\n",
+    "\n",
+    "PyC2 = openmc.Material(name='PyC2')\n",
+    "PyC2.set_density('g/cm3', 1.87)\n",
+    "PyC2.add_element('C', 1.0)\n",
+    "PyC2.add_s_alpha_beta('c_Graphite')\n",
+    "\n",
+    "SiC = openmc.Material(name='SiC')\n",
+    "SiC.set_density('g/cm3', 3.2)\n",
+    "SiC.add_element('C', 0.5)\n",
+    "SiC.add_element('Si', 0.5)\n",
+    "\n",
+    "graphite = openmc.Material()\n",
+    "graphite.set_density('g/cm3', 1.1995)\n",
+    "graphite.add_element('C', 1.0)\n",
+    "graphite.add_s_alpha_beta('c_Graphite')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To actually create individual TRISO particles, we first need to create a universe that will be used within each particle. The reason we use the same universe for each TRISO particle is to reduce the total number of cells/surfaces needed which can improve performance by a factor of two over using unique cells/surfaces in each."
```

Comment:
I think that was for a particular case but it probably varies quite a bit depending on the model, packing fraction, etc. I'll make this a little more general.